docs/apache-airflow/installation.rst

```diff
@@ -70,27 +70,24 @@ Installing via ``Poetry`` or ``pip-tools`` is not currently supported. If you wi
 using those tools you should use the :ref:`constraint files <installation:constraints>`  and convert them to appropriate
```

Comment:
It won't let me comment on line 25 cuz there's no change but I wanted to let you know there's an I in PyPI that should be capitalized :)